src/components/hero/hero.test.tsx

```diff
@@ -1,16 +1,41 @@
 import CampaignHero from "@/components/hero/campaignHero";
 import { render, screen } from "@testing-library/react";
-import { props } from "../../../__tests__/data/homepageProps";
 describe("Campaign Hero", () => {
-  it("renders the Campaign Hero", () => {
-    render(
-      <CampaignHero
-        featuredItem={props.featuredItem}
-        numberOfDigitizedItems="863,848"
-      />
-    );
-    expect(screen.getByRole("heading")).toHaveTextContent(
-      "Explore 863,848 items digitized from The New York Public Library's collections."
-    );
+  it("renders the SkeletonLoader component instead of the Campaign Hero", () => {
+    (global as any).fetch = jest.fn(() =>
+      Promise.resolve({
+        status: 200,
+        json: () => Promise.resolve({}),
+      })
+    ) as jest.Mock;
+
+    const { container } = render(<CampaignHero imageID="5273165" />);
+
+    expect(screen.getByTestId("hero-skeleton-loader")).toBeTruthy();
   });
+
+  // it("renders the Campaign Hero", () => {
+  //   (global as any).fetch = jest.fn(() =>
+  //     Promise.resolve({
+  //       status: 200,
+  //       json: () =>
+  //         Promise.resolve({
+  //           featuredItem: {
+  //             imageID: "5273165",
+  //             imageSrc: imageURL("5273165"),
+  //             uuid: "3a2cced0-12ca-0133-7a6d-58d385a7bbd0",
+  //             title:
+  //               "Latona Giving Birth to Apollo and Diana on the Island of Delos",
+  //             href: `https://qa-digitalcollections.nypl.org/items/3a2cced0-12ca-0133-7a6d-58d385a7bbd0`,
+  //           },
+  //           numberOfDigitizedItems: "863,848",
+  //         }),
+  //     })
+  //   ) as jest.Mock;
+  //   render(<CampaignHero imageID="5273165" />);
```

Comment:
The latest updates are great. You're still missing this commented out test and a similar one for the swim lanes component. If you're not going to test it right now, fine... but make sure you add/fix this test at some point before prod deployment. Let me know what you plan on doing and I can approve. The comment below is along the lines to fixing this.

Reply:
The commented out test fails because `render(<CampaignHero imageID="5273165" />)` returns the skeleton loader... so I'm not quite sure how to get this to pass. We can discuss this on Monday? Hopefully QA will be unblocked by then.